### PR TITLE
unsubscribe from newsletters you already subscribe to

### PR DESCRIPTION
**Pull request by laurieboyes**
screen shot 2016-07-04 at 12 04 57

### CODE REVIEW

File: views/temp-alert-delete-confirm.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="mypage">
+	{{>header}}
+
+	<div class="o-grid-container">
+		<div class="o-grid-row">
+			<div data-o-grid-colspan="12">
+				<p>TODO: fix n-notification core experience</p>
```
Comment (phamann) [inline views/temp-alert-delete-confirm.html]: Slightly cautious of having a `TODO` in the actual HTML markup 😬

Comment (laurieboyes) [inline views/temp-alert-delete-confirm.html]: Yeah fair enough

File: server/controllers/alerts/delete.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	const userId = res.locals.userId;
+	const listId = req.body.listId;
+
+	unsubscribeUserFromNewsletter(userId, listId)
+		.then(() => {
+			const viewModel = {
+				layout: 'wrapper',
+				trackablePageName: 'alerts'
+			};
+			res.render('temp-alert-delete-confirm', viewModel);
```
Comment (phamann) [inline server/controllers/alerts/delete.js]: This should be a redirect. I.e. FORM -> DELETE -> REDIRECT -> GET

File: server/app.js [hunk 3]
```diff
@@ -99,9 +99,12 @@ authRoute.get('/following/?:userId?', require('./controllers/following/default')
 authRoute.get('/saved-articles/?:userId?.?:extension(rss|json)?', require('./controllers/saved-articles'));
 authRoute.get('/explore/?:userId?', require('./controllers/explore'));
 authRoute.get('/average-push-frequency/?:userId?', require('./controllers/average-push-frequency'));
-authRoute.get('/alerts/?:userId?', require('./controllers/alerts'));
 authRoute.get('/product-tour-get-started', require('./controllers/product-tour-get-started'));
 
+//Emails and Alerts
+authRoute.get('/alerts/?:userId?', require('./controllers/alerts/read'));
+authRoute.post('/alerts/?:userId?/unsubscribe', require('./controllers/alerts/delete'));
```
Comment (phamann) [inline server/app.js]: Think the API interactions and the view for these journeys should be seperate

**Pull request merged**